src/amo/actions/reviews.js

```diff
@@ -431,3 +432,30 @@ export const clearAddonReviews = ({
     payload: { addonSlug },
   };
 };
+
+type SetLatestReviewParams = {|
+  review: ExternalReviewType | null,
+  addonId: number,
+  addonSlug: string,
+  versionId: number,
+  userId: number,
+|};
+
+export type SetLatestReviewAction = {|
+  type: typeof SET_LATEST_REVIEW,
+  payload: SetLatestReviewParams,
+|};
+
+export const setLatestReview = (
+  params: SetLatestReviewParams,
```

Comment:
Is this a new pattern you would like to introduce? Usually, we list all the properties of that object.